modules/core/04-channel/keeper/packet.go

```diff
@@ -22,15 +22,15 @@ import (
 func (k Keeper) SendPacket(
 	ctx sdk.Context,
 	channelCap *capabilitytypes.Capability,
-	packet exported.PacketI,
+	srcPort string,
+	srcChannel string,
+	timeoutHeight clienttypes.Height,
+	timeoutTimestamp uint64,
+	data []byte,
 ) error {
```

Comment:
The sequence should be returned

Reply:
+1 please add this. we're currently doing this on a fork, but it would be easier for us to maintain if this was the default behavior https://github.com/Stride-Labs/ibc-go/pull/1/files

Reply:
Thanks for the feedback @asalzmann! I've noted it on #1969. I'll see if we can get someone to pick it up relatively soon

Reply:
When this change is made, a test should be added to ensure the correct sequence is returned (and not the next sequence send value)